docs/sbom-tool-arguments.md

```diff
@@ -21,69 +21,70 @@ Actions
-    FollowSymlinks (-F)      If set to false, we will not follow symlinks while traversing the build drop folder. Default is set to 'true'.
-    ManifestInfo (-mi)       A list of the name and version of the manifest format that we are using.
 
   Generate -options - Generate a SBOM for all the files in the given build drop folder, and the packages in the components path.
 
-    Option                                    Description
-    BuildDropPath (-b)                        Specifies the root folder of the drop directory containing the final build artifacts (binaries and executables) for which the SBOM file will be generated.
-                                              This is the directory where the completed build output is stored.
-    BuildComponentPath (-bc)                  Specifies the folder containing the source code and components used to build the binary. This is where the tool will look for the individual components and
-                                              packages that are part of the build process.
-    BuildListFile (-bl)                       The file path to a file containing a list of files one file per line for which the SBOM file will be generated. Only files listed in the file will be included in
-                                              the generated SBOM.
-    ManifestDirPath (-m)                      The path of the directory where the generated SBOM files will be placed. A folder named '_manifest' will be created at this location, where all generated SBOMs
-                                              will be placed. If this parameter is not specified, the files will be placed in {BuildDropPath}/_manifest directory.
-    PackageName (-pn)                         The name of the package this SBOM represents. If this is not provided, we will try to infer this name from the build that generated this package, if that also
-                                              fails, the SBOM generation fails.
-    PackageVersion (-pv)                      The version of the package this SBOM represents. If this is not provided, we will try to infer the version from the build that generated this package, if that also
-                                              fails, the SBOM generation fails.
-    PackageSupplier (-ps)                     Supplier of the package that this SBOM represents.
-    DockerImagesToScan (-di)                  Comma separated list of docker image names or hashes to be scanned for packages, ex: ubuntu:16.04, 56bab49eef2ef07505f6a1b0d5bd3a601dfc3c76ad4460f24c91d6fa298369ab.
-    AdditionalComponentDetectorArgs (-cd)     Additional set of arguments for Component Detector.  An appropriate usage of this would be a space-delimited list of `--key value` pairs, representing command-line arguments. See the component-detection repository for a list of valid arguments (https://github.com/microsoft/component-detection/blob/main/docs/detector-arguments.md).
```

Comment:
Looks like #499 changed the readme without changing the help text. I'll fix this.